moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -142,11 +142,8 @@ void MotionPlanningFrame::clearScene()
   {
     ps->getWorldNonConst()->clearObjects();
```

Comment:
How would I trigger this to test it? I don't see a service call nor a button called "Clear" in the Motion Planning GUI.

Reply:
Ah, I found it. It's the tiny button with the gray tab/X.